com.chartboost.mediation.demo/Assets/Demo/Demo.cs

```diff
@@ -30,8 +34,11 @@ public class Demo : MonoBehaviour
     public InputField bannerPlacementInputField;
     public Dropdown bannerSizeDropdown;
     public Dropdown bannerLocationDropdown;
-    private ChartboostMediationBannerAd _bannerAd;
-    private bool _bannerAdIsVisible;
+    private ChartboostMediationUnityBannerAd _bannerAd;
+    public Dropdown horizontalAlignmentDropdown;
+    public Dropdown verticalAlignmentDropdown;
```

Comment:
mixed ordering of public/private. Types not grouped.

Reply:
Done 👍